jkube-kit/jkube-kit-spring-boot/src/test/java/org/eclipse/jkube/springboot/watcher/SpringBootWatcherIntegrationTest.java

```diff
@@ -129,15 +130,19 @@ void withAllRequirementsShouldStartWatcherProcess() throws Exception {
       && new File(command[0]).exists()
       && command[1].equals("-cp")
       && command[2].contains(
-      absolutePath("spring-boot-lib.jar") + File.pathSeparator + absolutePath("spring-boot-devtools.jar"))
+      realPath("spring-boot-lib.jar") + File.pathSeparator + realPath("spring-boot-devtools.jar"))
       && command[3].equals("-Dspring.devtools.remote.secret=this-is-a-test")
       && command[4].equals("org.springframework.boot.devtools.RemoteSpringApplication")
       && command[5].matches("^http://localhost:\\d+$")
     ));
   }
 
-  private String absolutePath(String jar) {
-    return project.resolve("target").resolve(jar).toFile().getAbsolutePath();
+  private String realPath(String jar) {
```

Comment:
is it possible to inline this?

Reply:
I don't know how that would help. But the problem is that method `Path.toRealPath` throws an IO exception, and inlining the method can make handling the error a bit harder and the code less readable.

Reply:
@rohanKanojia Could you please explain what do you mean by inlining this method? because that does not look trivial to me

Reply:
Sorry, I just saw your comment. This was just a nitpick. I think it's fine this way as well.